src/graphql/validation/rules/lone_anonymous_operation.py

```diff
@@ -20,9 +20,8 @@ def __init__(self, context: ASTValidationContext):
 
     def enter_document(self, node: DocumentNode, *_args: Any) -> None:
         self.operation_count = sum(
-            1
+            isinstance(definition, OperationDefinitionNode)
```

Comment:
This makes use of the fact that bool is an int subtype in Python and `1 == True`.

Yes, we can do that.